news/resolvelib.vendor.rst

```diff
@@ -0,0 +1 @@
+Upgrade resolvelib to 1.1.0
```

Comment:
We should probably call out that this improves correctness, but may introduce worse performance in certain situations. I'm being hand-wavy with my wording as I assume you can come up with something better.

Reply:
Updated the news item, and seperately I think I will make `ResolutionTooDeep` and point users to a specific pip issue where I can collect scenarios where this is happens.